src/binsparse/_io/api.py

```diff
@@ -15,7 +16,7 @@
 from binsparse._types import GroupTypes
 
 
-def read(group: GroupTypes) -> sparse.spmatrix:
+def read(group: GroupTypes, struct: str | None = "scipy") -> gb.Matrix | sparse.spmatrix:
```

Comment:
Could this be `class` instead of `struct`?

Reply:
I would also consider passing the class itself instead of a string, but not super important rn

Reply:
Sure. Passing the class itself is a bit tricky for the type hinting. Would we be passing `sparse` or the specific matrix ( `sparse.csr_matrix`, etc)? Also if the latter, then that change to a specific module being passed as an arg makes having the format useless in some cases as we're passing the exact structure we would want, right?

For the simplicity of the PR I kept the string passing